### PR TITLE
Removed unnecessary changes and kept required changes intact

### PR DESCRIPTION
**Pull request by gtpeddi94**
Created a new branch and made changes to it. Corrected unnecessary edits in previous push and pushed valid modifications again.

### CODE REVIEW

File: configurations/Controllership_field_config.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
+
```
Comment (fitzworkhub) [inline configurations/Controllership_field_config.yaml:1]: You're missing Controllership_config.yaml

File: scripts/generate_export.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import os
 import sys
+from indico.client.client import IndicoClient
```
Comment (fitzworkhub) [inline scripts/generate_export.py:3]: Do you use this anywhere? If not remove it.

File: scripts/generate_export.py [hunk 3]
```diff
@@ -9,7 +10,7 @@
 from solutions_toolkit.indico_wrapper import IndicoWrapper
 
 
-USAGE_STRING = "USAGE: python3 generate_export path/to/configuration_file"
+USAGE_STRING = "python3 generate_export path/to/configuration_file"
```
Comment (fitzworkhub) [inline scripts/generate_export.py:13]: Keep the USAGE bit, it's a clue to the user that it's an instruction. (Might be a good idea to use arg parse here, just a thought for refactoring).

**Review by fitzworkhub — CHANGES_REQUESTED** [review]
Last couple of changes. Make them on this branch and push. 
You can see one of the issues from creating a new branch and recreating changes is that you may end up forgetting stuff.

Thanks for pushing these changes though, they're necessary if we want to use this script more globally at CW.

File: configurations/Controllership_field_config.yaml (hunk 1 above)
Comment (gtpeddi94) [reply configurations/Controllership_field_config.yaml:1]: I deliberately removed it. It was an unnecessary file. I'll push separate config.yaml files for each use case later if required.

**Review by fitzworkhub — APPROVED** [review]

**Comment by fitzworkhub** [comment]
Approved. Checkout the `main` branch on your local and do `git pull`

**Comment by gtpeddi94** [comment]
Sure. After checking out main it says the branch is ahead of main by 2 commits. Did something not go through?

Gnana Teja Peddi
Machine Learning/Intelligent Data Capture Developer
gnanateja.peddi@ashlingpartners.com<mailto:gnanateja.peddi@ashlingpartners.com>
(312) 998-9545

**Comment by fitzworkhub** [comment]
What happens when you run `git pull` then `git status`? Should be ok

**Comment by gtpeddi94** [comment]
After git pull it said “merge made by recursive strategy” and after git status it said “nothing to commit.”

Gnana Teja Peddi
Machine Learning/Intelligent Data Capture Developer
gnanateja.peddi@ashlingpartners.com<mailto:gnanateja.peddi@ashlingpartners.com>
(312) 998-9545

**Comment by fitzworkhub** [comment]
Perfect! You should be good to go, you should see all the changes you made. 

Thanks for pushing this stuff!

**Comment by gtpeddi94** [comment]
Thanks for bearing with me on a Friday night. Happy weekend!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**